contract-examples/package.json

```diff
@@ -27,23 +28,19 @@
   },
   "license": "BSD-3-Clause",
   "scripts": {
+    "build": "rm -rf dist/ && npx tsc -b",
```

Comment:
Why are we using `npx` if we have `typescript` installed locally?

Reply:
well I always find `npx` as the "safest" option. I believe it will run local script if found. But you're correct we probably don't need it here as we declared typescript as a dependency.